README.md

```diff
@@ -307,4 +335,31 @@ custom:
                     Fn::GetAtt: [MyStream, Arn]
 ```
 
+> It is important that the mapping template will return a valid `application/json` string
+
 Source: [How to connect SNS to Kinesis for cross-account delivery via API Gateway](https://theburningmonk.com/2019/07/how-to-connect-sns-to-kinesis-for-cross-account-delivery-via-api-gateway/)
+
+#### SNS
+
+Similar to the [Kinesis](#kinesis-1) support, you can customize the default request mapping templates in `serverless.yml` like so:
+
+```yml
+custom:
+  apiGatewayServiceProxies:
+    - kinesis:
+        path: /sns
+        method: post
+        topicName: { 'Fn::GetAtt': ['SNSTopic', 'TopicName'] }
+        request:
+          template:
+            application/json:
+              'Fn::Join':
```

Comment:
can you change this to use `Fn::Sub` instead? e.g.

```
'Fn::Sub': "Action=Publish&Message=$util.urlEncode('This is a fixed message')&TopicArn=$util.urlEncode('${MyTopic}')"
```

Reply:
Of course

Reply:
good point, then maybe mention the `serverless-pseudo-parameters` as well? `Fn::Join` is just so unwildy and hard to follow

Reply:
Updated the readme. I referenced`serverless-cloudformation-sub-variables`.
`serverless-pseudo-parameters` lets you do the same?

Reply:
yup, the two kinda do the same thing

Reply:
Should I change it to `serverless-pseudo-parameters`?

Reply:
nah, it's fine, leave it, they do the same thing